client/app/components/users/UserEdit.jsx

```diff
@@ -101,22 +108,34 @@ export class UserEdit extends React.Component {
     });
   };
 
-  renderBasicInfoForm() {
+  renderUserInfoForm() {
     const { user } = this.state;
-    const formFields = [
+
+    const formFields = reject([
       {
         name: 'name',
         title: 'Name',
         type: 'text',
         initialValue: user.name,
+        required: true,
       },
       {
         name: 'email',
         title: 'Email',
         type: 'email',
         initialValue: user.email,
+        required: true,
       },
-    ].map(field => ({ ...field, readOnly: user.isDisabled, required: true }));
+      (currentUser.isAdmin && currentUser.id !== user.id) ? {
+        name: 'group_ids',
+        title: 'Groups',
+        type: 'select',
+        mode: 'multiple',
+        options: this.state.groups,
+        initialValue: this.state.groups.filter(group => includes(user.groupIds, group.value))
+          .map(group => group.value),
+      } : null,
+    ], isNull).map(field => ({ ...field, readOnly: user.isDisabled }));
```

Comment:
Considering that it might take some time to load the groups, maybe have some loading state for the data? I know Ant's Select components have support for this, so it's only a matter of communicating this between the components.

Reply:
Cool, locally it's already done, I also added a "Loading..." placeholder. (better than a whole blank select imo)